test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+
+npm run lint
+
+function cleanup() {
+	docker stop deltatest || true
+	docker rm --volumes deltatest || true
+}
+trap cleanup EXIT
+
+docker build -f Dockerfile.test -t docker-delta-test .
```

Comment:
Also worth noting that running the tests leaves images behind and needs manual cleanup afterwards. Just a heads up because this might be an issue with hosts like Jenkins.